composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "lenrsmith/evapi-php",
```

Comment:
@lenrsmith looks like this and the homepage link below should be updated to `exavault/evapi-php`, correct?